index.js

```diff
@@ -324,7 +324,12 @@ module.exports = function (log, indexesPath) {
       const fieldStart = opData.seek(buffer)
       if (~fieldStart) {
         const buf = bipf.slice(buffer, fieldStart)
-        addToPrefixMap(index.map, seq, buf.length ? safeReadUint32(buf) : 0)
+        const offset = opData.prefixOffset ? opData.prefixOffset : 0
```

Comment:
:warning: `offset` is a name already taken by the argument to this function. We should use something like `prefixOffset` instead. Preferably also in the arg name of `safeReadUint32` also to make it easier for people to read the code (suppose they are doing a grep for "offset")

Reply:
With my suggestion of default function arg above in `safeReadUint32`, we can just directly pass `opData.prefixOffset` as argument to `safeReadUint32`